packages/medusa/src/types/common.ts

```diff
@@ -1,7 +1,7 @@
 import { Transform } from "class-transformer"
 import { IsOptional, IsString, IsNumber, IsDate } from "class-validator"
 import { Type } from "class-transformer"
-import { IsType } from "../utils/is-type"
+import { transform } from "../utils/validators/date-transform"
```

Comment:
Would probably rename this, as it is confusing how this is almost identical to `Transform`. Maybe just call it `dateTransform` ?

Reply:
sure